portfolio.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>David Leiva</title>
+  </head>
+  <body>
+
+    <!-- Main navigation bar or toolbar -->
+    <header>
+      <label class="logo">My Logo</label>
+      <nav>
```

Comment:
- [ ] Please notice the decoration image inside the figma design. We need to have that same image on our webpage. As a front-end engineer, we are expected to clone the given template into a webpage with all its content and style. 

### Figma design 
![image](https://user-images.githubusercontent.com/69077061/228934625-eeb27f76-34bc-4701-953a-432450a2010a.png)

### Your design 
![image](https://user-images.githubusercontent.com/69077061/228934713-1b539049-42a9-442c-a1b4-39afe9b9cd52.png)